abb_resources/urdf/inertial_macro.xacro

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <!--
+    NOTE: the macro defined here is copied from Universal Robots/ur_description package
+    https://github.com/ros-industrial/universal_robot/blob/1c719dd15b79ae83a5bd6c1be46d712cbbf129ec/ur_description/urdf/inc/ur_common.xacro#L11-L19
+  -->
```

Comment:
this is already much better, but ideally we'd copy the license header or (summerised/shortened) license text here.

This is important, as `abb_resources` is a copy of the upstream package from `ros-industrial/abb` and [that's `Apache-2.0`](https://github.com/sam-xl/abb_ros2/blob/d2c4b8ac37584574805c82c16a223d0e19c140c3/abb_resources/package.xml#L16), while `ur_description` [is `BSD` (`3-Clause`)](https://github.com/ros-industrial/universal_robot/blob/4f3c776b3661e5b6e3f25a961382084873e2d3cc/ur_description/package.xml#L18).

Compatible, but different licenses.